File: app/src/main/java/com/example/shiranpeer/burritolocator/util/StringUtil.java
package com.example.shiranpeer.burritolocator.util;

public class StringUtil {
    public static String dollarBuilder(int priceLevel) {
        StringBuilder dollars = new StringBuilder();
        for (int i = 0; i < priceLevel; i++) {
            dollars.append("$");
        }

        return dollars.toString();
    }
}

File: app/src/main/java/com/example/shiranpeer/burritolocator/util/StringUtil.kt
package com.example.shiranpeer.burritolocator.util

object StringUtil {
    fun dollarBuilder(priceLevel: Int): String {
        val dollars = StringBuilder()
        for (i in 0 until priceLevel) {
            dollars.append("$")
        }

        return dollars.toString()
    }
}
